osu.Game/Rulesets/Scoring/ScoreProcessor.cs

```diff
@@ -40,6 +40,11 @@ public partial class ScoreProcessor : JudgementProcessor
         /// </summary>
         public readonly BindableLong TotalScore = new BindableLong { MinValue = 0 };
 
+        /// <summary>
+        /// The current score without <see cref="ScoreMultiplier"/> applied.
+        /// </summary>
+        public readonly BindableDouble BaseScore = new BindableDouble { MinValue = 0 };
```

Comment:
I'm still really lost as to why all this is exposing `BaseScore` and `ScoreMultiplier` when we already _had_ an implementation that was completely constrained to `ScoreProcessor` in https://github.com/ppy/osu/pull/23510#discussion_r1193185796.

I want this score multiplier disable stuff to touch as little places as possible. It should be in `ScoreProcessor`, and it should be in the realm database so that replays can be played back correctly (arguable depending on expectations and requirements that are still not clear to me on that front). Nothing else should need to concern itself about any of this.

Reply:
I reset implementation back to the way it was.

Hmm, I last time I checked locally saved score doesn't contin total score in replay frames. Only final score is handled currently.